src/main/java/stsjorbsmod/JorbsMod.java

```diff
@@ -456,4 +458,14 @@ public void receiveStartGame() {
             }
         }
     }
+
+    @Override
+    public void receiveCardUsed(AbstractCard c) {
+        SkillsPlayedThisActSaveData sp =
```

Comment:
Same here; I don't think `BaseMod.getSaveFields` is necessary, the `skillsPlayed` field is static